integration/spark-common-cluster-test/src/test/scala/org/apache/carbondata/cluster/sdv/generated/BatchSortLoad1TestCase.scala

```diff
@@ -1,292 +0,0 @@
-
```

Comment:
SDVSuites.scala is still pointing to this file. Need to modify that file also ?

Reply:
yeah, it also needs to remove 'PartitionTestCase'.